viewer/package.json

```diff
@@ -57,12 +60,14 @@
     "worker-plugin": "^3.2.0"
   },
   "dependencies": {
-    "@cognite/potree-core": "^1.1.1",
-    "@cognite/sdk": "^2.13.0",
     "comlink": "^4.1.0",
     "gl-matrix": "^3.1.0",
     "glslify": "^7.0.0",
-    "glslify-import": "^3.1.0",
+    "glslify-import": "^3.1.0"
+  },
+  "peerDependencies": {
```

Comment:
Why are these being made into peerDependencies?

Reply:
Three and SDK are peerDependencies because the user will decide what version they should be. They are not bundled anymore after moving them to peerDeps (i.e. no duplicated three.module.js). potree-core is maybe a stretch, what I really want it to be is an optionalDependency...

Reply:
Would not an optionalDependency work for all these then? I think `peerDependencies` are more for packages that are like plugins than ones that have optional dependencies like we do.